### DISCUSSION TITLE
Welcome to principia Discussions!

### DISCUSSION THREAD

**Discussion by sdac** [Bithack/principia#25] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by electroscientistX** [comment]
OFFICIALLY AWESOME!!!
I hope everyone is aware of https://principia-web.se/ by now, it will be awesome if we can play levels directly (without changing 0x4 hex code for level version identifier or something)

Happy to see principia finally getting open source, will love to contribute :)
